package.json

```diff
@@ -1,19 +1,19 @@
 {
   "name": "vtex",
-  "version": "2.97.0",
+  "version": "2.98.0-beta.5",
   "description": "The platform for e-commerce apps",
-  "main": "lib/index.js",
-  "bin": "lib/cli.js",
+  "bin": "bin/run",
   "scripts": {
+    "prepack": "rm -rf build && yarn tsc -b && OCLIF_COMPILATION=true yarn oclif-dev manifest",
```

Comment:
No need for `prepack` script for now - the CI already builds before releasing to npm